Source/Scene/ModelAnimationCollection.js

```diff
@@ -100,7 +100,7 @@ define([
      * @param {ModelAnimationLoop} [options.loop=ModelAnimationLoop.NONE] Determines if and how the animation is looped.
      * @returns {ModelAnimation} The animation that was added to the collection.
      *
-     * @exception {DeveloperError} Animations are not loaded.  Wait for the {@link Model#readyToRender} event.
+     * @exception {DeveloperError} Animations are not loaded.  Wait for the {@link Model#readtPromise} to resolve.
```

Comment:
Typo here and throughout, `Model#readtPromise`.  I would have just fixed it, but I wanted to ask if the `{@link Model#readtPromise}` markup in the `DeveloperError` below resolves to a link in our exception display?